src/styles/_catalog.scss

```diff
@@ -1,10 +1,26 @@
 .catalog {
-  display: flex;
+  display: grid;
+  grid-template: 300px / repeat(4, 200px);
```

Comment:
300px and 200px are repeated too oftem. Make variables for them.

Reply:
thanks a lot for the review